AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Actions/SignUp/InitiateSignUp.swift

```diff
@@ -48,15 +49,21 @@ struct InitiateSignUp: Action {
             return
         }
 
-        let input = SignUpInput(username: signUpEventData.username,
-                                password: signUpEventData.password,
-                                attributes: signUpEventData.attributes,
-                                environment: environment)
+
         logVerbose("\(#fileID) Starting signup", environment: environment)
 
         Task {
             let event: SignUpEvent
             do {
+                let asfDeviceId = try? await CognitoUserPoolASF.asfDeviceID(
```

Comment:
can we use `try` otherwise we can miss the error thats being thrown.  ..

Reply:
thanks for catching this. Updated